lib/elixir/src/elixir_parser.yrl

```diff
@@ -905,7 +905,9 @@ extract_identifier({Kind, _, Identifier}) when
 
 build_nested_parens(Dot, Args1, {Args2Meta, Args2}, {BlockMeta, Block}) ->
   Identifier = build_parens(Dot, Args1, {[], []}),
-  Meta = BlockMeta ++ Args2Meta ++ ?meta(Identifier),
+  % Take line and column meta from the call target node
+  LocationMeta = [{Key, Value} || {Key, Value} <- ?meta(Identifier), Key == line orelse Key == column],
```

Comment:
I personally think that a `lists:filter` reads more like a filter here rather than a comprehension whose only job is to filter (if I’m getting that right):

```suggestion
  LocationMeta = lists:filter(fun({Key, _}) -> Key == line orelse Key == column end, ?meta(Identifier))
```

Reply:
Sounds good :)